### ISSUE TITLE
GutterX,GutterY in % terms

### ISSUE THREAD

**Issue by way2nikhil** [AshesOfOwls/jquery.shapeshift#50]
Can the gutterx and gutterY be provided in %? .
I dont want to harcode .
Secondly the columnwidths can also be provided in % terms , so it will be easy to add up gutter space as required by plugin

**Comment by AshesOfOwls** [comment]
Dynamic gutters is definitely a feature I want to include for version 3.0 (which i think will be a july release date).

There are some complications with doing percent widths which mainly has to do with recalculating the grid every time. It starts to get very process heavy when doing that especially if there are a lot of child elements.

I will definitely consider that part for 3.0 but I might not get to it. There have been a lot of feature requests recently and unfortunately this doesn't rank too high in terms of performance but I don't intend on abandoning this project any time soon so I will consider it for 4.0 if I don't get to it. I tend to have a 4 month version release schedule fyi.

**Comment by AshesOfOwls** [comment]
This is now in 3.0 and is working quite well. Should be released within the next month.

**Comment by way2nikhil** [comment]
thanks for the update !!
Can you share me the current build code  .I want to give it a try.

**Comment by AshesOfOwls** [comment]
Definitely, all the code is up on the public 3.0 branch so you should be able to find it in the dropdown list.

I don't really have any documentation yet for it so it may be a bit difficult to play with. I would suggest to look at the demos/basic.html page and to check out shapeshift.coffee to see what options you can initialize the program with.

For now the gutter can accept data like this:
Gutter: [20,20] // normal
Gutter: ['20%',20] // percentage gutter x
Gutter ['auto',20] // automatically determine gutter

The colwidth can be set to null, an integer, or '20%' for a percent width

**Comment by way2nikhil** [comment]
Hi mcpants,

Can you please check this issue and tell if its easily fixable or any quick
fix can be done? or if i can expect any fix in future releases.

Thanks,
regards
Nikhil
